### DISCUSSION TITLE
Cannot run tests

### DISCUSSION THREAD

**Discussion by izerozlu** [mockoon/mockoon#393] [Help] [answered]
I cannot seem to run tests with `npm run test`. 

I've been creating a PR for the #308 and I've finished the implementation. I've wanted to run the tests and add a few more for the feature but I cannot get it going. I've contributed before and I know that I've added tests before.

What I'm doing is building the application via `npm run build:dev` as mentioned in contributing guideline and after that running `npm run test`. But no Mockoon window gets running for testing. 

So naturally I receive 
>  "before all" hook for "Should wait for window to be ready"

for each test suite.

Any help?

**Comment by 255kb** [comment, marked as answer]
First a dumb question: is your branch up to date with master? All the tests dependencies has been updated and now everything is more stable. Also there was a bug under MacOS which has been fixed.

**Reply by izerozlu** [reply]
Yes I presume that I'm in sync with the master branch.
You can see my git tree below. If you spot a problem please let me know.

![Screen Shot 2020-12-29 at 13 36 16](https://user-images.githubusercontent.com/17386157/103277843-f0685680-49da-11eb-8f52-b8951a01842b.png)

**Reply by 255kb** [reply]
It looks ok.
Did you run an npm install? Cleaning your node modules and reinstalling could help as many dependencies were updated.

**Reply by izerozlu** [reply]
That did the trick, thanks!

**Reply by 255kb** [reply]
great 👍
